packages/jest-config/src/index.ts

```diff
@@ -267,6 +229,90 @@ This usually means that your ${chalk.bold(
   }
 };
 
+/**
+/**
+ * Reads the jest config, without validating them or filling it out with defaults.
+ * @param config The path to the file or serialized config.
+ * @param param1 Additional options
+ * @returns The raw initial config (not validated)
+ */
+export async function readInitialOptions(
+  config?: string,
+  {
+    packageRootOrConfig = process.cwd(),
+    parentConfigDirname = null,
+    readFromCwdInstead = false,
+    skipMultipleConfigError = false,
+  }: {
+    /**
+     * The package root or deserialized config (default is cwd)
+     */
+    packageRootOrConfig?: string | Config.InitialOptions;
+    /**
+     * When the `packageRootOrConfig` contains config, this parameter should
+     * contain the dirname of the parent config
+     */
+    parentConfigDirname?: null | string;
+    /**
+     * Indicates whether or not to read the specified config file from disk.
+     * When true, jest will read try to read config from the current working directory.
+     * (default is false)
+     */
+    readFromCwdInstead?: boolean;
+    /**
+     * Indicates whether or not to ignore the error of jest finding multiple config files.
+     * (default is false)
+     */
+    skipMultipleConfigError?: boolean;
```

Comment:
Does this need to be an option? Seems one should always fail of that's the case

Reply:
It was already supported by `readConfig`. Since `readConfig` is using `readInitialOptions` internally I thought I would add this feature to `readInitialOptions` as well. 

Removing this feature altogether might be a breaking change, right?